plugins/modules/win_shell.py

```diff
@@ -115,6 +115,11 @@
   ansible.windows.win_shell: C:\somescript.ps1
   args:
     no_profile: true
+
+- name: Run a PowerShell script with big5 encoding (fixing garbled text issue)
```

Comment:
I'm not sure there is really a need to document this for `win_shell`. PowerShell will be using the existing console encoding which we set to `utf-8`. Any executable that the `.ps1` script will call is subject to the encoding rules of PowerShell itself and shouldn't affect the `output_encoding_override` unless the script itself changes the `[Console]::OutputEncoding`.

Reply:
I'm a user for Taiwan, using Traditional Chinese.
Tested using Windows 2012 R2 as managed nodes.

Reply:
Thanks for sharing, I think this is the case because keytool will be writing directly to the console and not going through PowerShell's string encoding. Appreciate you sharing this use case as I wasn't aware that would have been a problem.